openapi_python_client/templates/model.pyi

```diff
@@ -14,7 +16,13 @@ class {{ model.reference.class_name }}:
     {{ property.to_string() }}
     {% endfor %}
 
-    def to_dict(self) -> Dict[str, Any]:
+    def to_dict(
+        self,
+        include: Optional[Set[str]] = None,
+        exclude: Optional[Set[str]] = None,
+        exclude_unset: bool = False,
+        exclude_none: bool = False,
```

Comment:
A couple notes here:
1. Are these here just in case someone wants them? Or was there a request specifically for this? I'd rather not add stuff to the API unless there's a demand for it since it slows things down a bit and might mean we have to breaking change it in the future.
2. When would we ever want to _not_ `exclude_unset`?

My instinct is to scratch these params and always skip values that are `UNSET`.

Reply:
The need originally spawned out of #228, but realistically theres no need atm as the separation between `required` and `nullable` fixes the issue I was trying to address